OneCommunityGlobal/HGNRest
# Jatin create backend for custom forms

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:
<img width="671" alt="Screenshot 2024-10-16 at 7 20 53 AM" src="https://github.com/user-attachments/assets/531541dc-917c-4472-b483-b8985394c558">


## Related PRS (if any):
This backend PR is not related to any frontend PR.
…

## Main changes explained:
- Create file src/controllers/formController.js and src/controllers/formController.spec.js .
- Create file src/routes/formRouter.js and src/controllers/formRouter.test.js .
- Create file src/models/forms.js and src/models/formResponse.js
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run build` to run this PR locally
3. Clear site data/cache
4. Use any API Testing tool for testing the API. (POSTMAN is recommended)

## Screenshots or videos of changes:
